src/screens/ContactScreen/index.js

```diff
@@ -105,17 +110,24 @@ const ContactScreen = ({navigation}) => {
   const handleCreateChannel = async () => {
     try {
       const mappingUserName = selectedUsers?.map((user) => user?.username).join(',');
+      const mappingUserId = selectedUsers?.map((user) => user?.user_id).join(',');
       let image = DEFAULT_PROFILE_PIC_PATH;
       if (selectedUsers.length === 1) {
         image = selectedUsers[0]?.profile_pic_path;
       }
       const dataSelected = {
         user: {
           name: mappingUserName,
-          image
+          image,
+          userId: mappingUserId
         }
       };
-      createSignChat(followed, dataSelected);
+
+      if (sourceScreen === ANONYMOUS) {
+        createAnonymousChat(dataSelected);
+      } else {
+        createSignChat(followed, dataSelected);
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is no error handling for the `createAnonymousChat` and `createSignChat` functions. If these functions throw an error, it will not be caught and handled properly. Consider wrapping them in a try-catch block.
```diff
-       if (sourceScreen === ANONYMOUS) {
-         createAnonymousChat(dataSelected);
-       } else {
-         createSignChat(followed, dataSelected);
-       }
+       try {
+         if (sourceScreen === ANONYMOUS) {
+           await createAnonymousChat(dataSelected);
+         } else {
+           await createSignChat(followed, dataSelected);
+         }
+       } catch (error) {
+         console.error('Error creating chat:', error);
+       }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->